docs/installation/tks/Installing_TKS_Clone.md

```diff
@@ -0,0 +1,92 @@
+Installing TKS Clone
+====================
+
+Overview
+--------
+
+This page describes the process to install a TKS subsystem as a clone of an existing TKS subsystem.
+
+Exporting Existing System Certificates
+--------------------------------------
+
+Export the existing system certificates (including the certificate chain) into a PKCS #12 file, for example:
+
```

Comment:
This comment applies to entire doc.  In general, unless it doesn't matter, it would be a good idea to tell people where they are expected to run the command from. This is particularly useful for people who are not familiar with the command (like myself).

Reply:
I think all of our tools can be run from anywhere. If that's not the case please let me know, we need to fix it. Input and output files can be specified using absolute paths or paths relative to the current directory, just like most other tools.

By default the `pki` CLI will use the NSS database in `~/.dogtag/nssdb`, and it can be changed with the `-d <path>` option. By default the `pki-server` CLI will use the NSS database in `/etc/pki/pki-tomcat/alias`, and it can be changed with the `-i <instance>` option. But in general we want people to use these tools with the default values so it will be a lot easier to use, so we don't want to mention these details unless it's necessary. Just like Firefox, we don't really care about the NSS database location. We only care about file location only when we're uploading/downloading something.